.github/workflows/tests-lints.yml

```
@@ -21,11 +21,13 @@ jobs:
           target: wasm32-unknown-unknown
           override: true
 
-      - name: Run cargo-tarpaulin
-        uses: actions-rs/tarpaulin@v0.1
+      - name: Run tests
+        uses: actions-rs/cargo@v1
```

actionlint (1.7.4)

the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)